src/index.js

```diff
@@ -163,6 +164,7 @@ function createConfig(options, entry, format) {
 			input: entry,
 			external,
 			plugins: [
+				flow({ all: true }),
```

Comment:
I'm not sure that `all: true` should be the default here

Reply:
@thejameskyle what are the reasons that `all` shouldn't be default?